engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -706,8 +705,16 @@ private void run(
             .enableStaticAnalysis(enableStaticAnalysis)
             .executionEnvironment(executionEnvironment != null ? executionEnvironment : "live")
             .warningsLimit(warningsLimit)
-            .options(options)
-            .build();
+            .options(options);
+
+    if (inspect) {
+      options.put("inspect", "");
```

Comment:
I realized that when one uses `--inspect` and hit `Debug.breakpoint` statement we actually _hit the breakpoint twice_. Once in Chrome Dev Tools and once in REPL. I believe it is better to have just a single debugger running. As such I am proposing to **disable debug server** when `--inspect` option is used.

Reply:
That definitely makes sense.